xformers/ops/fmha/common.py

```diff
@@ -99,6 +99,43 @@ def normalize_bmhk(self) -> Tuple[int, ...]:
             self.value = self.value.unsqueeze(2)
         return output_shape
 
+    def validate_bmhk(self) -> None:
```

Comment:
if you move this to the beginning of the function, I would maybe call it `validate_inputs`?